axelrod/strategies/cycler.py

```diff
@@ -77,6 +78,8 @@ def strategy(self, opponent):
 
 
 class CyclerDC(Cycler):
+
+    name = 'Cycler DC'
```

Comment:
Yes. the dynamic assignment only works at instance level and the API needs a unique id for each class.